### PR TITLE
CSRF protection

### PR DESCRIPTION
**Pull request by domachine**
**HEADS UP**: We still need a solution for a case when the first request in the application is a `POST` / `PATCH` / `PUT` request.

### CODE REVIEW

File: app/lib/app/shared/api/backend.js [hunk 1]
```diff
@@ -162,6 +162,14 @@ export async function getAdvisories() {
   return advisories
 }
 
+export async function getAboutInfo() {
+  const response = await new APIRequest(new Request('/api/v1/about'))
```
Comment (domachine) [inline app/lib/app/shared/api/backend.js:166]: Make sure to use the `CSRFAPIRequest`

File: app/lib/app/App.js [hunk 2]
```diff
@@ -42,6 +42,8 @@ export default function App({ secvisogramPage }) {
         .then(
           (result) => {
             setUserInfo(result)
+            // make sure the first call is a GET so that the CSRF cookie is set
+            api.backend.getAboutInfo().then(() => {})
```
Comment (domachine) [inline app/lib/app/App.js:46]: Return or await api promise here to track its progress.

File: app/lib/app/shared/api/backend.js (hunk 1 above)
Comment (domachine) [reply app/lib/app/shared/api/backend.js:166]: Ah no. Forget this.

File: app/lib/app/App.js (hunk 2 above)
Comment (cloeser) [reply app/lib/app/App.js]: I now handle that in CsrfApiRequest and only make the call if no cookie is set.

**Pull request merged**